lib/execute.js

```diff
@@ -25,6 +25,13 @@ const os = require('os');
 const crypto = require('crypto');
 
 /**
+ * @summary A temporary directory to store scripts
+ * @type {String}
```

Comment:
Would it be worth adding a note here on why XDG_RUNTIME_DIR is necessary?

Reply:
Done!